src/modules/ui/app/app.html

```diff
@@ -18,33 +18,33 @@
                     <recipe-hello-for-each></recipe-hello-for-each
                     ><recipe-hello-iterator></recipe-hello-iterator>
                 </template>
-                <template if:true={navigationItems.composition.visible}>
+                <template lwc:if={navigationItems.composition.visible}>
```

Comment:
@adityanaag3 you'll want to use `lwc:elseif` for these conditions since only one navigation item is visible at any time.

Reply:
@pozil Done.